examples/spark/spark-gluster/spark-master.yaml

```diff
@@ -0,0 +1,26 @@
+kind: Pod
+apiVersion: v1
+metadata: 
+  name: spark-master
+  labels: 
+    name: spark-master
+spec: 
+  containers: 
+    - name: spark-master
+      image: gcr.io/google_containers/spark-master
+      privileged: true
```

Comment:
This field doesn't exist. Use --validate=true (now the default) when creating the resource. privileged is a member of securityContext.